shards/egui/src/containers/selectable.rs

```diff
@@ -172,70 +184,128 @@ impl Shard for Selectable {
       }
     };
 
-    // Draw frame and contents
-    let inner_response = with_possible_panic(|| {
-      ui.push_id(id, |ui| {
-        Frame::group(ui.style()).stroke(stroke).show(ui, |ui| {
-          util::activate_ui_contents(context, input, ui, &mut self.parents, &mut self.contents)
+    // ui.interact(rect, id, sense)
+
+    let drag_data = self.drag_data.get();
+    let can_drag = !drag_data.is_none();
+
+    let mut err: Option<&str> = None;
+    ui.push_id(id, |ui| -> Result<(), &str> {
+      let mut body = |ui: &mut egui::Ui| {
+        // Draw frame and contents
+        with_possible_panic(|| {
```

Comment:
I guess we try remove those after merge?

Reply:
I didn't fix the stack thing yet